src/pages/PageCallManage/Styles.ts

```diff
@@ -0,0 +1,69 @@
+import { StyleSheet } from 'react-native'
+
+import g from '../../components/variables'
+import CustomColors from '../../utils/CustomColors'
+
+const styles = StyleSheet.create({
+  video: {
+    position: 'absolute',
+    top: 40, // Header compact height
+    bottom: 0,
+    left: 0,
+    right: 0,
+    backgroundColor: 'black',
```

Comment:
same here

Reply:
I am modifying this file for the next ticket, It will be fixed in the next PR.